apps/meteor/client/views/admin/moderation/helpers/ReportReasonCollapsible.tsx

```diff
@@ -0,0 +1,24 @@
+import { Box, Button } from '@rocket.chat/fuselage';
+import { useTranslation } from '@rocket.chat/ui-contexts';
+import React, { useState } from 'react';
+import type { ReactNode } from 'react';
+
+const ReportReasonCollapsible = ({ children }: { children: ReactNode }) => {
+	const [isOpen, setIsOpen] = useState(false);
+	const t = useTranslation();
+
+	const toggle = () => setIsOpen((prev) => !prev);
```

Comment:
Consider using `useToggle` from `@rocket.chat/fuselage-hooks` instead of `useState`.

Reply:
Thank you so much for the review. I will use that. I did not know there was such a hook; interesting.